mne/filter.py

```diff
@@ -2435,6 +2435,18 @@ def float_array(c):
     )
 
 
+def _check_resamp_noop(sfreq, o_sfreq, rtol=0):
```

Comment:
`rtol=1e-6` seems like a more reasonable choice here. It should work for `float32` pretty well